Doc/library/mimetypes.rst

```diff
@@ -88,6 +88,10 @@ behavior of the module.
    Specifying an empty list for *files* will prevent the system defaults from
    being applied: only the well-known values will be present from a built-in list.
 
+   If *files* is not specified or is ``None`` the internal data structure is completely
```

Comment:
Since `files` is already set to `None` you don't have to explicitly call out what happens if you don't specify an argument.